package.json

```diff
@@ -18,7 +18,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```

Comment:
Why this is in tests?

Reply:
No this isn't for tests I added this because while deploying to GitHub pages it was deploying README instead of the frontend. So to fix it I added these lines.

Reply:
but this should be defined in deploy.yml not test.yml, no?

Reply:
Sorry, but I have defined this in package.json file.

Reply:
Ahh OK then, this is good